.github/workflows/backport.yml

```
@@ -23,7 +23,7 @@ jobs:
               )
             )
         steps:
-            - uses: tibdex/backport@v2
+            - uses: tibdex/backport@2e217641d82d02ba0603f46b1aeedefb258890ac
```

suggest including comments saying which version the hashes are supposed to correspond to, example:


```suggestion
            - uses: tibdex/backport@2e217641d82d02ba0603f46b1aeedefb258890ac   # v2
```